skins/10h16/templates/contact_form.html.twig

```diff
@@ -38,6 +38,21 @@
                   {$ util.form_error('email', errors) $}
                 </div>
 
+                <div class="input-container">
+                  <label for="">Thema</label>
+                  <select id="category" name="category" class="category">
```

Comment:
Looks like the contents of the `<select>` can be represented by using a for statement.
https://twig.symfony.com/doc/2.x/tags/for.html

Reply:
Hmm, using a `for n in 1..7` that will lead to errors about missing keys or missing output when the number of topics changes. But you're right, iteration would be good here. We should do a followup where we: 

1) move the categories out of `messages.json` into their own JSON file in `i18n/de_DE/data/contact_categories.json`
2) Add an array parameter for the categories to `GetInTouchHtmlPresenter`. That array should be passed as `contact_categories` on to the template in the `render` call of the presenter.
3) In `FunFunFactory::newGetInTouchHtmlPresenter` load the language-appropriate data file (see https://github.com/wmde/FundraisingFrontend/blob/master/src/Factories/FunFunFactory.php#L476-L477 for inspiration, maybe that can be extracted into a private `loadLocalizedData` method)
4) Use `for topic in contact_topics` in the templates.

@aniansson I hope my description is sufficient, would you like to try doing it?